images/versioned/r-ver.md

```diff
@@ -31,7 +31,7 @@ Compared to `r-base`,
   (the `r-base` stack gets the latest R version as a binary from Debian unstable).
 - The only platforms available are `linux/amd64` and `linux/arm64`
   (arm64 images are experimental and only available for `rocker/r-ver` 4.1.0 or later).
-- Set [the Posit Public Package Manager (RStudio Package Manager, RSPM)](https://packagemanager.rstudio.com)
+- Set [the Posit Public Package Manager (P3M, a.k.a RStudio Package Manager, RSPM)](https://packagemanager.rstudio.com)
```

Comment:
You may want to update the URL to the posit.co one too.

(Do they really brand it as P3M?  Ouch.)

Reply:
> (Do they really brand it as P3M? Ouch.)

Yes, this appears to be used in the recently released Posit's packages. (rocker-org/rocker-versioned2#638)